v2/azure-arm.yaml

```diff
@@ -814,22 +826,35 @@ status:
 #
 # Available 'type' modifiers:
 #
-#   $renamedTo: <string>  // Gives the name this type will have in the next 
-#                         // version of the resource
+# $export: <bool>
+#     Requests that support for this resource type be generated.
+#     Automatically includes all other types required for this resource
+#     If not specified, false is assumed.
+#     Only valid for resource types.
+#
+# $exportAs: <string>
+#     Requests that this resource type be exported with the specified
+#     identifier, allowing poorly named types to be renamed.
+#     Implies $export: true
+#     Only valid for resource types.
+#
+# $nameInNextVersion: <string>
+#     Gives the name this type will have in the next version of the resource.
 #
 # Available 'property' modifiers:
 #
-#   $armReference: <bool> // Indicates whether this property is an ARM 
-#                         // reference to another resource. 
-#                         // Set to 'true' when the property is a reference
-#                         // Set to 'false' to disable our heuristics
+# $armReference: <bool>
+#     Indicates whether this property is an ARM reference to another resource. 
+#     Set to 'true' to confirm the property is an ARM reference.
+#     Set to 'false' to disable our heuristics if a property is incorrectly 
+#     identified as an ARM reference.
 #
-#   $isSecret: <bool>     // Specifies that the property references a secret
-#                         // Set to `true` to integrate with ASO custom secret
-#                         // management.
+# $isSecret: <bool>
+#     Specifies that the property references a secret.
+#     Set to `true` to integrate with ASO custom secret management.
 #
-#   $renamedTo: <string>  // Gives the name this property will have in the 
-#                         // next version of the resource
+# $nameInNextVersion: <string>
+#     Gives the name this property will have in the next version of the resource
```

Comment:
Will have, or _has_? Is it a directive to change the name or a directive that the name _is_ a particular value? It sorta sounds like the former but I think you actually mean the latter.

Reply:
Clarified.